changelogs/unreleased/gh-8650-add-is_sync-option.md

```diff
@@ -0,0 +1,6 @@
+## feature/box
```

Comment:
The docbot only mentions `box.atomic`, but you added it to `box.begin` and `box.commit` too. Please, document all three. Same for the netbox changes.

Reply:
### **What have we agreed:**

-  We leave `is_sync` in all three places (begin, commit, atomic) with the only `true` option.
    - We throw an error to the user when he specifies `is_sync != true` or any other value like `is_sync = "some string"`.

-  For users, we document only `atomic`. Documentation for the guru about `begin` and `commit` will be described only in the commit message.

- `is_sync`  should also appear in the `iproto` protocol (`iproto_begin` and `iproto_commit`).